src/web/middleware/db.rs

```diff
@@ -16,6 +16,7 @@ use crate::web::middleware::sentry::{queue_report, report};
 use crate::web::{
```

Comment:
This middleware is no longer part of the project, so the file can be deleted. Most of the code has been moved into `DbTransactionPool`.